README.md

```diff
@@ -303,26 +303,45 @@ This project uses a [Zenodo](https://zenodo.org) generated DOI. The badge at the
 
 Many ISIS applications require ancillary data. For example, calibration applications require flat files to do flat field corrections, and map projection applications require DTMs to accurately compute intersections. Due to its size, this data is stored in a separate directory called the ISIS Data Area. Any location can be used for the ISIS Data Area, the software simply requires that the ISISDATA environment variable is set to its location.
 
-### Structure of the ISIS3 Data Area
+### Structure of the ISIS Data Area
 
 Under the root directory of the ISIS Data Area pointed to by the ISISDATA environment variable are a variety of sub-directories. Each mission supported by ISIS has a sub-directory that contains mission specific processing data such as flat files and mission specific SPICE. There are also data areas used by more generic applications. These sub-directories contain everything from templates to test data.
 
-### Size of the ISIS3 Data Area
+### Versions of the ISIS Data Area
```

Comment:
Should you swap the order of these two sections? 'Structure of the ISIS Data Area' mentions the ISISDATA variable, then in 'Versions of the ISIS Data Area' you talk about  the difference between the name of the isis data variable expected by different versions of ISIS.

Reply:
Or you could just say "pointed to by the ISISDATA/ISIS3DATA environment variable" or "pointed to by the isis data environment variable"